ldcoolp/curation/main.py

```diff
@@ -160,7 +161,8 @@ def workflow(article_id, url_open=False, browser=True, log=None,
         pw.reserve_doi()
 
         # Retrieve data and place in 1.ToDo curation folder
-        pw.download_data()
+        if not metadata_only:
+            pw.download_data()
```

Comment:
Based on what the intent of `metadata_only` boolean flag is, `download_data` method should be triggered, but to only retrieve/write the files metadata.

Also, `metadata_only` should be set as an instance variable within the `PrerequisiteWorkflow` class. This will gives us the greatest flexibility for other methods.